services/uploads/serverless.yml

```diff
@@ -29,6 +29,7 @@ provider:
             - s3:PutObjectTagging
             - s3:PutObjectVersionTagging
             - s3:ListBucket
+            - s3:DeleteObject
```

Comment:
This is an OK change if it's needed by the lambda now.  
However, I have noticed what looks like a typo in a line nearby.  You didn't change the line, but it should be fixed.
See a little bit below, in the policy block for s3:ListBucket ...
There are two Resources specified.  Those resources are arns of buckets.
You see how one has a trailing /* and one doesn't?  That's wrong.  Neither should have a trailing "/*".
This has likely caused issues for the clamdefs bucket, as ListBucket effectively has not been allowed.